### PR TITLE
CNV-9975 [1913948] Clarify size of pvc specification when uploading

### PR DESCRIPTION
**Pull request by bgaydosrh**
Please peer review, merge and CP to  *enterprise-4.6*, *enterprise-4.7*, and *enterprise-4-8*

Jira: https://issues.redhat.com/browse/CNV-9975

Bob

### CODE REVIEW

File: modules/virt-uploading-local-disk-image-dv.adoc [hunk 1]
```diff
@@ -64,6 +64,8 @@ $ virtctl image-upload dv <datavolume_name> \ <1>
 ====
 * If you do not want to create a new data volume, omit the `--size` parameter and
 include the `--no-create` flag.
+* When uploading a disk image to a PVC, the PVC size must be larger than the size of the uncompressed virtual disk.
+====
```
Comment (sjhala-ccs) [inline modules/virt-uploading-local-disk-image-dv.adoc:68]: @bgaydosrh I think you need to remove this extra `====` here. That might be the source of error.

Comment (bgaydosrh) [reply modules/virt-uploading-local-disk-image-dv.adoc]: Thanks. I didn't see the third note :-(

**Review by kgoldbla — APPROVED** [review]

**Comment by kgoldbla** [comment]
LGTM

**Review by sjhala-ccs — COMMENTED** [review]
LGTM!

**Pull request merged**